registry.json

```diff
@@ -5218,5 +5218,19 @@
    "telegram": "VegasAda",
    "operatorName": "",
    "github": ""
+ },
+ {
+   "ticker": "TROPC",
+   "poolId": "7447913be564ff4e2505fb21ca5de980517f3b1575e006ce2d754b74",
+   "social": {
+     "twitter": "TropicPoolADA",
+     "website": "https://tropicpool.io",
+     "facebook": "",
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "TropicPoolADA",
```

Comment:
Please change this to the telegram username of an individual and not a group or channel

Reply:
This is [updated](https://github.com/SinglePoolAlliance/Registration/pull/630/files#diff-c1ccab6b761f7c3ef53302c329215f605bb24c73aa16fb75bc4da712926e972bR5287). Thanks!